dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScopeTest.kt

```diff
@@ -103,9 +105,22 @@ internal class RumResourceScopeTest {
     var fakeServerOffset: Long = 0L
 
     private lateinit var fakeEventTime: Time
+    var fakeSourceResourceEvent: ResourceEvent.Source? = null
+    var fakeSourceErrorEvent: ErrorEvent.ErrorEventSource? = null
+
+    @Mock
+    lateinit var mockRumEventSourceResolver: RumEventSourceResolver
 
     @BeforeEach
     fun `set up`(forge: Forge) {
+        fakeSourceResourceEvent = forge.aValueFrom(ResourceEvent.Source::class.java)
```

Comment:
shouldn't we wrap it in `aNullable` call as well?

Reply:
Good catch